### ISSUE TITLE
As a SO/CE I need to make sure that text placed over an image inside a hero unit has enough contrast to be readable

### ISSUE THREAD

**Issue by kevincrafts** [CuBoulder/express#828]
## Context

Placing text on top of images does not always result in something that is easy to read. While this could be solved with image editors, not all of our clients have access or the knowledge of how to use them.

This ticket adds an option of inserting an overlay on the image to make the text easier to read.

## Process

1. Create a Hero Unit block
2. Add a background image and some text
3. Check the "Add overlay to improve readability" box at the bottom of the design tab

## Expected result

Hero Unit should have an overlay that makes the text easier to read.

## Current result

This feature does not currently exist.

## Screenshots
![screen shot 2017-02-27 at 2 54 10 pm](https://cloud.githubusercontent.com/assets/349713/23381760/e483e804-fcfc-11e6-926d-914dcb774e24.jpg)

Before

![](https://media-api.atlassian.io/file/b14eafa2-dd9e-4e5a-bc79-13dd417d4afb/binary?token=eyJhbGciOiJIUzI1NiJ9.eyJpc3MiOiIxZjg1MzI1NS01NzIyLTRmYWUtYWQyYS02OWUxZjc1OTc2YjYiLCJhY2Nlc3MiOnsidXJuOmZpbGVzdG9yZTpmaWxlOmIxNGVhZmEyLWRkOWUtNGU1YS1iYzc5LTEzZGQ0MTdkNGFmYiI6WyJyZWFkIl19LCJleHAiOjE0ODgyMzMxOTUsIm5iZiI6MTQ4ODIzMjUzNX0.kwf5DVZqSaTGjmtHN5qv9kaOJ3OOophnOBKSduxkPq8&client=1f853255-5722-4fae-ad2a-69e1f75976b6&name=Screen+Shot+2017-01-28+at+10.44.06+AM.jpg)

After

![](https://media-api.atlassian.io/file/e5b3dbc6-0a59-453c-88b3-7f264187b5d4/binary?token=eyJhbGciOiJIUzI1NiJ9.eyJpc3MiOiIxZjg1MzI1NS01NzIyLTRmYWUtYWQyYS02OWUxZjc1OTc2YjYiLCJhY2Nlc3MiOnsidXJuOmZpbGVzdG9yZTpmaWxlOmU1YjNkYmM2LTBhNTktNDUzYy04OGIzLTdmMjY0MTg3YjVkNCI6WyJyZWFkIl19LCJleHAiOjE0ODgyMzMxOTksIm5iZiI6MTQ4ODIzMjUzOX0.uScQ_c7waqLsUP6zp398K6uaO6HRrHIYaV-PaPSo4vc&client=1f853255-5722-4fae-ad2a-69e1f75976b6&name=Screen+Shot+2017-01-28+at+10.44.18+AM.jpg)

**Comment by ekellagher** [comment]
Documentation on Web Central:
https://www.colorado.edu/webcentral/tutorial/v2/hero-unit-block

**Comment by cathysnider** [comment]
Verified: I re-tested this functionality; the hero units are looking good at all Size Priorities and all Sizes. The different text colors -- White and Black -- properly have the correct overlay color -- Black and White.  Additionally, changing the Block Theme did not have an adverse affect on the text within the block -- it remained readable.
